app/components/select/hint.njk

```diff
@@ -0,0 +1,64 @@
+{% set title = 'Select with hint text' %}
+{% from 'components/select/macro.njk' import select %}
+{% extends 'example.njk' %}
+
+{% block content %}
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-two-thirds">
+      {{ select({
+        "id": "location",
+        "name": "location",
+        "label": {
+          "text": "Choose location"
+        },
+        "hint": {
+          "text": "This can be different to where you went before"
+        },
+        "items": [
+          {
+            "value": "choose",
+            "text": "Choose location",
+            "selected": true
```

Comment:
Worth moving this to be `value: "choose"` at the top level, now we have that option?